stylus/utilities/bgcolor.styl

```diff
@@ -7,3 +7,16 @@ for color in keys(colors)
         background-color: colors[color] !important // @stylint ignore
 
     global('.u-bg-' + color, '$bg-color-' + color)
+
+$bg-primary-light
+    background-color: var(--primaryColorLightBackground) !important // @stylint ignore
```

Comment:
I'm wondering about those `!important` everywhere. Is this the right approach, or should we just clean it all up and make it work without the `!important` (if it's even possible, I don't know if it's ever been debated)?

Reply:
Utility classes have !important and it is not a problem since they should override everything.

I think we could debate this if it is a problem for you but I do not think it is the role of this PR.